app/assets/javascripts/shared/datatable/main.js

```diff
@@ -94,6 +94,20 @@ class DradisDatatable {
             text: '<i class="fa fa-tags fa-fw"></i>Tag<i class="fa fa-caret-down fa-fw"></i>',
             buttons: this.setupTagButtons()
           },
+          {
```

Comment:
This is polluting all data-tables for something we'll use exactly once.

You can extract this to the  app/assets/javascripts/shared/datatable/state.js and only add it if the DT needs it. See JIRA for an example.